src/tiledimage.js

```diff
@@ -1324,22 +1329,22 @@ function compareTiles( previousBest, tile ) {
 }
 
 function drawTiles( tiledImage, lastDrawn ) {
-    var i,
-        tile = lastDrawn[0];
-
-    if ( tiledImage.opacity <= 0 ) {
-        drawDebugInfo( tiledImage, lastDrawn );
```

Comment:
I ditched that considering that with #859 we are not coming here if opacity is <=0.